test/utils.jl

```diff
@@ -48,4 +48,45 @@
   @test uvrotation(V2(1,0), V2(0,1)) ≈ T[0 -1; 1 0]
   @test uvrotation(V2(0,1), V2(1,0)) ≈ T[0 1; -1 0]
   @test uvrotation(V3(1,0,0), V3(0,1,0)) ≈ T[0 1 0; -1 0 0; 0 0 1]
+
+  @testset "Point in mesh 3D" begin
```

Comment:
no need to create a testset at this moment. we can move all the sideof implementations to a separate file later.

Reply:
so just remove testset but keep in this file? sure.